hid-xpadneo/Makefile

```diff
@@ -1,15 +1,13 @@
-KERNEL_SOURCE_DIR := /usr/src/linux
-LD := ld.bfd
+KERNEL_UNAME  ?= $(shell uname -r)
+KERNEL_BUILD := /lib/modules/$(KERNEL_UNAME)/build
```

Comment:
There's a patch in my queue already for this but it didn't work reliably across different environments. But I can push it so you could rebase your work and see if you still need changes.

Reply:
?

Reply:
I'll push my changes later this day. Your patch confirms, that I should really push it and no longer let it sit in my local queue:

https://github.com/kakra/xpadneo/commit/216f094052fd0c7015b3c0421a45dc2f5cc958df

It currently uses the `source` symlink but that's more an artifact of my local build environment. I'll push it with a `build symlink`.